.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "files.exclude": {
```

Comment:
This just prevents VS Code from populating (and visually making the navigator jump around) with the generated files.

Reply:
Yup, I use this locally in conjunction with the vscode extension "Toggle Excluded FIles" (https://marketplace.visualstudio.com/items?itemName=eamodio.toggle-excluded-files)

It is pretty handy. Possibly something to add to the recommended extensions?

Reply:
Your call. I wanna go ahead and close out this PR so I wasn't planning on adding any more extensions, but I say go for adding it after all this gets merged into master